### DISCUSSION TITLE
Restore Win+X PowerShell Shortcut

### DISCUSSION THREAD

**Discussion by MaidScientistIzutsumiMarin** [valinet/ExplorerPatcher#336] [Ideas]
The Win+X menu has, in the place of PowerShell or Command Prompt from Windows 10, instead shortcuts to Windows Terminal. This is fine for most people, I imagine, since Windows Terminal uses one of the two as its backend regardless. However, for people who don't want to have a separate app installed just to launch Windows Terminal, it would be nice to be able to just have direct links to PowerShell or Command Prompt there instead of Windows Terminal (I suppose just let people choose which of the three to display).

As a side note, I apologize if that is what _Show Command Prompt instead of PowerShell in Win+X menu_ is meant to do. Toggling that option and restarting File Explorer doesn't appear to have any effect on the Win+X menu. If it is an issue with that option, I will move this to issues.

**Comment by valinet** [comment]
Don't open an issue, as that's how Win+X works in Windows 11, apparently. They just disregard that setting and put Windows Terminal in there and that's it. The setting in EP just changes the corresponding registry entry that used to work in Windows 10. To add PowerShell or cmd or whatever back, use a Win+X menu editor utility.
